mercury/static/mercury/style.css

```diff
@@ -60,11 +60,11 @@
   
   body {
     margin: 0;
-    font-family: "Nunito", -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, "Helvetica Neue", Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol", "Noto Color Emoji";
+    font-family:"Helvetica Neue"; 
```

Comment:
It's a good practice to have more than one font listed in case the primary one is unavailable. What if you just bump "Helvetica Neue" to the front of the list?

Reply:
Extremely valid-- thanks!